Makefile

```diff
@@ -415,7 +415,7 @@ run-smoke:
 	sleep 20
 	sed -i -e 's|HOST_TO_REPLACE|$(POSTCODE_ENDPOINT)|g' $(APPLICATION_TEST_DIR)/contract/environment/postcode_smoke.postman_environment.json
 
-	if [ $(PROFILE) == pd ] || [ $(PROFILE) == dmo ] || [ $(PROFILE) == stg ]
+	if [ $(PROFILE) == pd ] || [ $(PROFILE) == dmo ] || [ $(PROFILE) == stg ] || [ $(PROFILE) == dev ]
```

Comment:
Please remove dev, this allows for deployment of postcode api in nonprod independently of fuzzy search